### PR TITLE
Added NODE_SIDE_TYPE flag to helm chart

### PR DESCRIPTION
**Pull request by PeterChung241**

### CODE REVIEW

**Comment by madhavajay** [comment]
@PeterChung241 Include in `helm` values.yaml template. Check `helm.py`, then test with `helm install --arg`

**Comment by shubham3121** [comment]
Example: `helm install <node_name> <repo> --set nodeSideType=low/high`

File: packages/grid/devspace.yaml [hunk 1]
```diff
@@ -267,6 +269,8 @@ deployments:
                 value: "${VERSION_HASH}"
               - name: NODE_TYPE
                 value: "${NODE_TYPE}"
+              - name: NODE_SIDE_TYPE
+                value: "${NODE_SIDE_TYPE}"
```
Comment (rasswanth-s) [inline packages/grid/devspace.yaml]: NIT: We could remove this from frontend as it not use node_side_type env variable

**Comment by rasswanth-s** [comment]
Command: **Low Side** 🔅 

```sh
helm install syft-gateway ./syft --set node.settings.nodeType="gateway" --set node.settings.nodeName="un-petlab" --set node.settings.nodeSideType="low"
```

Screenshot 2023-12-05 at 8 16 13 AM

**Comment by rasswanth-s** [comment]
Command: **High Side** 🔆 

```sh
helm install syft-domain ./syft --set node.settings.nodeType="domain" --set node.settings.nodeName="canada" --set node.settings.nodeSideType="high"
```
Screenshot 2023-12-05 at 8 28 26 AM

File: packages/grid/devspace.yaml (hunk 1 above)
Comment (rasswanth-s) [reply packages/grid/devspace.yaml]: Resolved in [b3e40f1](https://github.com/OpenMined/PySyft/pull/8293/commits/b3e40f1d8959443ab0513a61280890be44988d3e)

**Review by rasswanth-s — APPROVED** [review]
Well Done @PeterChung241  @shubham3121

**Pull request merged**